.github/workflows/publish-chart.yaml

```diff
@@ -32,7 +31,7 @@ jobs:
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.4.0
         with:
-          charts_dir: charts
-          charts_repo_url: https://helm.resoto.com
+          charts_dir: someengineering
+          charts_repo_url: https://helm.some.engineering
```

Comment:
@kushthedude `charts_dir` should have been updated in #23, correct?

Reply:
Yes, that's correct, I forgot to update it in the action. Thanks @TheCatLady